db/migrate/20211105110735_create_cards_decks.rb

```diff
@@ -5,8 +5,7 @@ def change
       t.bigint :deck_id, null: false
       t.timestamps
       
-      t.index :card_id
-      t.index :deck_id
+      t.index [:card_id, :deck_id], unique: true
```

Comment:
This is really bad. You should not change existing migrations from commits that were pushed to remote repositories.

Reply:
This will not deploy properly either.